mmv1/templates/terraform/examples/cloud_run_anthos.tf.erb

```diff
@@ -0,0 +1,38 @@
+data "google_client_config" "provider" {}
+
+data "google_container_cluster" "my_cluster" {
+  name     = "my-cluster"
+  location = "us-central1"
+}
+
+provider "kubernetes-alpha" {
+  host  = "https://${data.google_container_cluster.my_cluster.endpoint}"
+  token = data.google_client_config.provider.access_token
+  cluster_ca_certificate = base64decode(
+    data.google_container_cluster.my_cluster.master_auth[0].cluster_ca_certificate,
+  )
+}
+
+resource "kubernetes_manifest" "test-configmap" {
+  provider = kubernetes-alpha
+
+  manifest = {
+    "apiVersion" = "serving.knative.dev/v1"
+    "kind" = "Service"
+    "metadata" = {
+      "name" = "helloworld-go"
+      "namespace" = "default"
+    }
+    "spec" = {
+      "template" {
```

Comment:
I don't think this works? At least when I try to run this config it complains it is invalid syntax. Have you been able to set this up successfully?

I also have kubernetes-alpha issues, but that's probably unrelated

Reply:
Sorry, there are a few typos in the step but I have tried it now.